TumeKyouen/View/StoneButton.m

```diff
@@ -29,7 +29,7 @@ - (id)initWithOptions:(int)size state:(int)defaultState {
             forControlEvents:UIControlEventTouchDown];
     }
 
-    return self;
+     return self;
```

Comment:
ERROR: Core Foundation/Objective-C
Returning 'self' while it is not set to the result of '[(super or self) init...]'